opentelemetry-sdk/src/opentelemetry/sdk/resources/__init__.py

```diff
@@ -471,6 +473,19 @@ def detect(self) -> "Resource":
         )
 
 
+class HostResourceDetector(ResourceDetector):
+    """
+       The HostResourceDetector detects the hostname and architecture attributes.
```

Comment:
Looks like this file needs to be formatted. If you haven't already, feel free to look at the [contributor guide](https://github.com/open-telemetry/opentelemetry-python/blob/main/CONTRIBUTING.md#development) for suggestions on how to do that to get the lint tests to pass.

Reply:
I think it's actually failing though. Isn't it? :-) https://github.com/open-telemetry/opentelemetry-python/actions/runs/10918643219/job/30319599029?pr=4182#step:5:132

Reply:
I have already set pre-commit, and there was no error message when i submitted that file
finally i remove that black by myself